Core/Src/can_handler.c

```diff
@@ -85,6 +91,7 @@ void can1_callback(CAN_HandleTypeDef* hcan)
 		break;
 	case CAN_TEST_MSG:
 		serial_print("UR MOM \n");
+	case CAN_BMS_MONITOR:
```

Comment:
Hamza should confirm but i think wed want to queue the message into the queue you made and then monitor